File: swift/classTest.swift
class Person{
    var name: String
    var age: Int
    
    init(name: String, age: Int){
        self.name=name
        self.age=age
    }
    
    func say(){
        print(self.name, self.age)
    }
}

func test(){
    let a=Person(name: "Stanley", age:20)
    a.say()
}

File: kotlin/classTest.kt

class Person(var name:String, var age:Int){
    fun say(){
        println("$name $age")
    }
    fun say(words:String){
        println(words)
    }
}

fun test(){
    var a = Person("Stanley", 20)
    a.say()
    a.say("apple")
}